src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -218,4 +275,69 @@ class CallActor(key: CallKey, locallyQualifiedInputs: CallInputs, backend: Backe
   private def registerAbortFunction(abortFunction: AbortFunction): Unit = {
     self ! CallActor.RegisterCallAbortFunction(abortFunction)
   }
+
+  private def subscribe(subscriberActorRef: ActorRef, backendActor: ActorRef) = {
+    val subscription = Subscription[ActorRef](new ExecutionEvent, subscriberActorRef)
+    backendActor ! BackendActor.SubscribeToEvent[ActorRef](subscription)
+  }
+
+  // Currently this method will create a backend
+  private def createBackendActor(callObj: Call = call): ActorRef = {
+    val backendName: String = callObj.task.runtimeAttributes.attrs.get("backendName").headOption.map(_.head).getOrElse("")
+    val backendConfig = BackendConfiguration.apply()
+    val defaultBackendCfgEntry = backendConfig.getDefaultBackend()
+    val backendCfgEntry = backendConfig.getAllBackendConfigurations().filter(_.name.equals(backendName)).headOption.getOrElse(defaultBackendCfgEntry)
+    val task = buildTaskDescriptor(callObj)
+    DefaultBackendFactory.getBackend(backendCfgEntry.initClass, actorSystem, task)
+  }
+
+  /**
+    * Returns a TaskDescriptor, given the context of a task
+    *
+    * @return
+    */
+  private def buildTaskDescriptor(callObj: Call): TaskDescriptor = {
+    val name = callObj.fullyQualifiedName
+    val user = System.getProperty("user.name")
+    // Need Declarations, CallInputs, command template sequence
+    val cmdTemplateSeq = callObj.task.commandTemplate
+    val declarations = callObj.task.declarations
+    val runtimeAttributes = callObj.task.runtimeAttributes.attrs.map { case (k, v) => (k, v.head) }
+    TaskDescriptor(name, user, cmdTemplateSeq, declarations, s"${workflowDescriptor.name}-${workflowDescriptor.id}", locallyQualifiedInputs, callObj.task.outputs, runtimeAttributes)
+  }
+
+  private def useCachedIfPossible(backend: ActorRef): Future[Any] = Future {
+    if (workflowDescriptor.readFromCache) {
+      val cachedExecution = for {
+        hash <- (backend ? ComputeHash).mapTo[String]
```

Comment:
This should not be formulated as an ask.  We've just finished removing asks from the core of Cromwell as they are prone to timeouts in general, and this one is particularly likely to timeout on large inputs.